glama/start.sh

```
@@ -0,0 +1,10 @@
+#!/bin/bash
+set -e
+python /app/glama/echo-mcp.py &
+export ANON_PRINCIPAL_KEY="${ANON_PRINCIPAL_KEY:-glama-anon-principal-not-a-secret}"
+/app/glama/aisix --config /app/glama/config.yaml &
+for i in $(seq 1 50); do
```

Shellcheck (0.11.0)

[warning] i appears unused. Verify use (or export if used externally).

(SC2034)